scripts/deploy/iks/test/many-edges.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp import dsl, components
+import kfp
+
+PRINT_STR = """
+name: print
+description: print a message
+inputs:
+  - {name: msg, type: String}
+implementation:
+  container:
+    image: alpine:3.6
+    command:
+    - echo
```

Comment:
i don't think you can run with 'echo' as starting command, maybe something like `sh -c echo ....`?

Reply:
yes, it works. it took about 5 mins to complete the pipelinerun in my kind cluter :-D

And inside the script, it only verify if it can submit the run. it doesn't wait for the run to finish

Reply:
it looks good to me now